### ISSUE TITLE
Change TreatAs CLSID for better OPSEC

### ISSUE THREAD

**Issue by McCryptoFace** [nickvourd/COM-Hunter#2]
Currently, COM-Hunter installs TreatAs persistence with an easily fingerprintable `{DEADBEEF-BEEF-DEAD-BEEF-DEADBEEFDEAD}` CLSID.  Recommend changing this to a random CLSID, checking to ensure that the selected CLSID is not present before creating it.

**Comment by nickvourd** [comment]
Thanks a lot for your input... I will try to fix it... However COM-Hunter 2.0 will release this year (2023)... 😃

**Comment by McCryptoFace** [comment]
Oh sweet! That’s great to hear. I came across COM-Hunter just after doing
my RTO course, cool that you designed the tool out of that work! I looked
at COM-Hunter specifically because we were looking for tools to automate
that on my team, and your tool seems to be the best option around from what
I’ve seen. I still haven’t managed to make TreatAs persistence work in my
testing, but that could well be issues on my end. At any rate, thanks for
your work on this!

—Nick
